### PR TITLE
add option to disable patreon message

### PR DESCRIPTION
**Pull request by Oribuin**
please add this, it took me a whole 30 seconds to create this feature and id really appreciate it if you didnt make me waste a whole 30s on making this change

also it can be kinda annoying so to save the whining from people in #support, just merge it, we'd all be better off

### CODE REVIEW

**Comment by MertUnverdi** [comment]
close it dum, we won't remove that message (for some users should stay there)👍🏻

File: src/main/java/com/iridium/iridiumskyblock/listeners/PlayerJoinQuitListener.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public void onJoin(PlayerJoinEvent event) {
                 PlayerUtils.sendBorder(player, island)
         );
 
-        if (player.isOp()) {
+        if (player.isOp() && IridiumSkyblock.getInstance().getConfiguration().patreonMessage) {
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/listeners/PlayerJoinQuitListener.java:29]: This code can be simplified
```suggestion
        if (player.isOp() && true) {
```

**Comment by Oribuin** [comment]
> close it dum, we won't remove that message (for some users should stay there)👍🏻

i will find your family and bonk you upside the noggin bruv

**Comment by Oribuin** [comment]
the build failed because Esophose is moving the rosewood repo, I'm not a bad developer I promise

**Review by dlsf — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/listeners/PlayerJoinQuitListener.java (hunk 1 above)
Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/listeners/PlayerJoinQuitListener.java:29]: LOOOL

**Comment by PeachesMLG** [comment]
You know I really liked having this because it would annoy everyone, but fine >:C

**Pull request merged**